### ISSUE TITLE
IRIs are created by default

### ISSUE THREAD

**Issue by EnnoMeijers** [pldn/LDWizard#153]
With the changes introduced in 3.0.0 it looks like all data is transformed into IRIs by default. This should of course only be done when configured as a refinement option. 

For example:

Input data: 
header: titel
cell: "een momument in een neogotieke bouwstijl"

becomes:
<https://data.netwerkdigitaalerfgoed.nl/3c06c6/def/titel> <https://data.netwerkdigitaalerfgoed.nl/3c06c6/def/een%20momument%20in%20een%20neogotieke%20bouwstijl>

The config file used was:
```
import WizardConfig, { PrefixEntry } from "@pldn/ldwizard/types/WizardConfig";
import { getUriOfSearchTerm } from "./termennetwerk";

// @ts-ignore
import img from "./ndelogo.png";

// @ts-ignore
import favIcon from "./favicon.png";

// @ts-ignore
import homePage from "./homePage.md";

const wizardConfig: WizardConfig = {
  appName: "LDWizard - Erfgoed",
  defaultBaseIri: "https://data.netwerkdigitaalerfgoed.nl/",
  primaryColor: "#0a3dfa",
  secondaryColor: "#172a59",
  homepageMarkdown: homePage,
  publishOrder: ["download"],
  icon: img,
  favIcon: favIcon,
  dataplatformLink: "https://data.netwerkdigitaalerfgoed.nl/",
  repositoryLink: "https://github.com/netwerk-digitaal-erfgoed/LDWizard-ErfgoedWizard",
  getAllowedPrefixes: async () => {
    const response = await fetch("https://api.data.netwerkdigitaalerfgoed.nl/datasets/ld-wizard/sdo/prefixes");
    if (response.ok) {
      const prefixes: PrefixEntry[] = await response.json();
      return prefixes;
    }
    console.error("Prefixes retrieval failed")
    return [];
  }
};
export default globalThis.wizardConfig = wizardConfig;
```

With csv data:

```
recordnummer,titel,bouwstijl
1,een momument in een neogotieke bouwstijl,neogotiek
2,een momument in een romaanse travertin bouwstijl,Romaanse travertin
```

**Comment by philipperenzen** [comment]
This has been resolved in versio 3.0.2:

By default LDWizard will now return a literal from the untransformed CSV data. If an IRI output is desired with the base IRI, an empty transformation without specifying `yieldsIri` or `yieldsLiteral` should be used for this column - [see this example in the runtimeConfig file](https://github.com/pldn/LDWizard/blob/a6bde15f87d679b9df989e9627ec5c9173784a48/webpack/runtimeConfig.ts#L42).

**Comment by EnnoMeijers** [comment]
Thanks, looks good, I will do some more testing later!
